modules/Meetings/language/en_us.lang.php

```diff
@@ -96,10 +96,11 @@
     'LBL_REMINDER_EMAIL_ALL_INVITEES' => 'Email all invitees',
     'LBL_EMAIL_REMINDER' => 'Email Reminder',
     'LBL_EMAIL_REMINDER_TIME' => 'Email Reminder Time',
-    'LBL_REMOVE' => 'rem',
+    'LBL_REMOVE' => 'Remove',
     'LBL_SCHEDULING_FORM_TITLE' => 'Scheduling',
     'LBL_SEARCH_BUTTON' => 'Search',
     'LBL_SEARCH_FORM_TITLE' => 'Meeting Search',
+    'LBL_SEND_BUTTON_KEY'=>'I',
```

Comment:
Is this used at all?    `'LBL_SEND_BUTTON_KEY'=>'I',`

Reply:
Doesn't look like it, it's from the initial commit. I've removed it here: #6282